### PR TITLE
feat: added code to allow conditiioning on audio

### PR DESCRIPTION
**Pull request by tpavankalyan**

### CODE REVIEW

File: training/run_parler_tts_training.py [hunk 1]
```diff
@@ -243,6 +244,8 @@ def main():
         }
         if data_args.description_column_name is not None:
             columns_to_keep["description_column_name"] = data_args.description_column_name
+        if data_args.speech_emb_column_name is not None:
```
Comment (tpavankalyan) [inline training/run_parler_tts_training.py]: not needed

File: training/run_parler_tts_training.py [hunk 2]
```diff
@@ -243,6 +244,8 @@ def main():
...
+            columns_to_keep["speech_emb_column_name"] = data_args.speech_emb_column_name
```
Comment (tpavankalyan) [inline training/run_parler_tts_training.py]: not needed

File: training/run_parler_tts_training.py [hunk 3]
```diff
@@ -386,10 +396,11 @@ def main():
         # Preprocessing the dataset.
         # We need to tokenize the texts.
 
-        def pass_through_processors(description, prompt, language):
+        def pass_through_processors(description, prompt, language, speech_emb):
```
Comment (tpavankalyan) [inline training/run_parler_tts_training.py]: not needed

File: training/run_parler_tts_training.py [hunk 4]
```diff
@@ -386,10 +396,11 @@ def main():
...
             batch = {}
             batch["input_ids"] = description_tokenizer(description.strip())["input_ids"]
-            if model_args.prompt_tokenizer_name == "ai4bharat/indictrans2-en-indic-1B":
+            batch[speech_emb_column_name] = speech_emb
```
Comment (tpavankalyan) [inline training/run_parler_tts_training.py]: not needed

File: training/run_parler_tts_training.py [hunk 5]
```diff
@@ -401,7 +412,7 @@ def pass_through_processors(description, prompt, language):
             vectorized_datasets = raw_datasets.map(
                 pass_through_processors,
                 remove_columns=next(iter(raw_datasets.values())).column_names,
-                input_columns=[description_column_name, prompt_column_name, "language"],
+                input_columns=[description_column_name, prompt_column_name, "language", speech_emb_column_name],
```
Comment (tpavankalyan) [inline training/run_parler_tts_training.py]: not needed

File: training/run_parler_tts_training.py (hunk 1 above)
Comment (Dhruv88) [reply training/run_parler_tts_training.py]: removed in latest commit

File: training/run_parler_tts_training.py (hunk 4 above)
Comment (Dhruv88) [reply training/run_parler_tts_training.py]: removed in latest commit

File: training/run_parler_tts_training.py (hunk 2 above)
Comment (Dhruv88) [reply training/run_parler_tts_training.py]: removed in latest commit

File: training/run_parler_tts_training.py (hunk 3 above)
Comment (Dhruv88) [reply training/run_parler_tts_training.py]: removed in latest commit

File: training/run_parler_tts_training.py (hunk 5 above)
Comment (Dhruv88) [reply training/run_parler_tts_training.py]: removed in latest commit

**Pull request merged**